packages/react-components/review/react-components.api.md

```diff
@@ -123,6 +123,9 @@ export type CustomMessagePayload = {
 // @public
 export const darkTheme: PartialTheme & CallingTheme;
 
+// @public (undocumented)
+export const defaultIdentifiers: Identifiers;
```

Comment:
Let's not export these. These are only required for tests, and should be defined in the tests directly.

By exporting we're adding them to API contract, for no value.

Reply:
This approach requires default values to allow components to work without being wrapped inside providers.